specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3334,46 +3354,48 @@
         ],
         "responses": {
           "200": {
-            "description": "Successfully deleted hostname binding."
+            "description": "Function state returned.",
+            "schema": {
+              "type": "string"
+            }
           },
-          "204": {
-            "description": "Hostname binding does not exist."
+          "404": {
+            "description": "Function with a name of {functionName} does not exist."
           }
         }
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/hybridConnectionNamespaces/{namespaceName}/relays/{relayName}": {
-      "get": {
+      },
+      "put": {
         "tags": [
           "WebApps"
         ],
-        "summary": "Retrieves a specific Service Bus Hybrid Connection used by this Web App.",
-        "description": "Retrieves a specific Service Bus Hybrid Connection used by this Web App.",
-        "operationId": "WebApps_GetHybridConnection",
+        "summary": "Set the enablement state for a function.",
+        "description": "Set the enablement state for a function.",
+        "operationId": "WebApps_UpdateFunctionState",
         "parameters": [
           {
             "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
             "name": "name",
             "in": "path",
-            "description": "The name of the web app.",
+            "description": "Site name.",
             "required": true,
             "type": "string"
           },
           {
-            "name": "namespaceName",
+            "name": "functionName",
             "in": "path",
-            "description": "The namespace for this hybrid connection.",
+            "description": "Function name.",
             "required": true,
             "type": "string"
           },
           {
-            "name": "relayName",
-            "in": "path",
-            "description": "The relay name for this hybrid connection.",
+            "name": "requestEnvelope",
+            "in": "body",
             "required": true,
-            "type": "string"
+            "schema": {
+              "type": "string"
```

Comment:
This is not an RPC compliant proxy resource schema.

Reply:
Right, thanks. As noted above, trying to determine the correct schema mapping for this. As in my response payload example above, on the PUT, the API expects the request input to be an ARM wrapped payload with properties, e.g.:

```javascript
{ properties: 'disabled' }
```

Once we determine the correct schema format for the GET operation above, the same applies here.